guides/doc-Content_Management_Guide/topics/QuickStart.adoc

```diff
@@ -0,0 +1,104 @@
+[[Syncing_Centos_Stream]]
+== Syncing/Creating {customcontenttitle}
+This chapter outlines how you can manage Centos Streams {customcontent} of different types to {Project}. We will over the following steps.
+
+* Create `Custom` Repositories to mirror content from the Centos Stream
+* Synchronize the Content
+* Setup `Activation Keys`
+* Register Hosts to consume this content.
+* Manage Hosts, install/update/or remove rpms on `Host` via `katello-agent`
+
+
+== Create repositories to synchronize
+
+As a first step we need to let {Project} discover available repositories in the Centos stream. We then pick the repositories to mirror and bundle them into a Product.
+
+.Procedure
+. In the {Project} web UI, navigate to *Content* > *Products*
+. Click *Repo Discovery*.
+. In the *Repository Type* field, choose *Yum Repositories*.
+. In the *URL to Discover* field, enter the Centos Stream url `http://mirror.centos.org/centos/8-stream`.
+. Click *Discover*.
+. Once the repository discovery process is complete you should be able to list of repositories found under that url.
+. Select `/AppStream/x86_64/os/` and `/BaseOS/x86_64/os/` repositories.
+. Click *Create Selected*. This will enable you to create the repositories. However before creation we need  to point this group of repositories to a new product.
+. In the *Product* field choose *New Product*
+. In the *Name* field enter *Centos Stream* or desired product name.
+. Click *Run Repository Creation*. This should successfully create 2 repositories under the given product.
+
+_Note_ while repo discovery provides a convenient way to enable/create multiple repositories in one go, you could manually create the product and repositories directly.
+
+== Examine the generated entities
+
+In this section we will examine the repositories we created.
+
+. Navigate to *Content* > *Products*
+. Select the *Centos Stream* product that we created.
+. Notice the 2 repositories with names *AppStream x86_64 os* and *BaseOS x86_64 os*
+. Click on *AppStream x86_64 os*. This should take you to the Repository Details page.
+. Notice that the *Upstream URL* is set to `http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/` and the *Type* is set to *yum*
+
```

Comment:
I suggest we delete this section and just mention they can inspect the repos if they want.